src/controller/ebike_app.c

```diff
@@ -203,15 +208,58 @@ static void ebike_control_motor (void)
 
   if (!ui8_boost_enabled_and_applied)
   {
-    // cadence percentage (in x256)
-    ui16_temp = (uint16_t) (map (((uint32_t) ui8_tmp_pas_cadence_rpm),
-           (uint32_t) 0,
+    /*
+     * In case of wheel speed > 0
+     * 	Human cadence has a minimum value of MIN_CADENCE_RPM.
+     * 	Human torque has minimum MIN_TORQUE_UNITS(32 * ADC_TORQUE_UNITS = 256 TORQUE_UNITS so 1/8).
+     * If stopped use old behavior.
+     */
+    if (ui16_wheel_speed_x10 > 0)
+    {
+    	/*
+    	 * Cadence
+    	 */
+    	ui8_human_min_cadence = MIN_CADENCE_RPM;
+    	ui8_human_cadence = ui8_tmp_pas_cadence_rpm + ui8_human_min_cadence;
+        ui8_human_cadence = ui8_human_cadence > configuration_variables.ui8_pas_max_cadence ? configuration_variables.ui8_pas_max_cadence : ui8_human_cadence;
+
+    	/*
+    	 * Torque
+    	 */
+        ui8_human_min_torque = MIN_ADC_TORQUE_UNITS + ui8_adc_torque_sensor_min_value;
+        ui8_human_torque = UI8_ADC_TORQUE_SENSOR + ui8_human_min_torque;
+        ui8_human_torque = ui8_human_torque > ui8_adc_torque_sensor_max_value ? ui8_adc_torque_sensor_max_value : ui8_human_torque;
+    }
+    else
+    {
+    	/*
+    	 * Cadence
+    	 */
+    	ui8_human_cadence = ui8_tmp_pas_cadence_rpm;
+    	ui8_human_min_cadence = 0;
+    	/*
+    	 * Torque
+    	 */
+    	ui8_human_torque = ui8_torque_sensor;
+    	ui8_human_min_torque = ui8_adc_torque_sensor_min_value;
+    }
+
+	// cadence percentage (in x256)
+    ui16_cadence_p = (uint16_t) (map (((uint32_t) ui8_human_cadence),
+           (uint32_t) ui8_human_min_cadence,
            (uint32_t) configuration_variables.ui8_pas_max_cadence,
            (uint32_t) 0,
            (uint32_t) 255));
+    // torque percentage (in x256)
+    ui16_torque_p = (uint16_t) (map (
```

Comment:
Did not reuse the torque calculation done in the specific function because it is used in more places and I do not want to change that logic. (ie torque lcd menu)